index.html

```diff
@@ -2824,11 +2824,11 @@ <h2 id="OfflineAudioCompletionEvent">
             </dl>
           </dd>
           <dt>
-            readonly attribute AudioContext context
+            readonly attribute BaseAudioContext context
           </dt>
           <dd>
             <p>
-              The <a><code>AudioContext</code></a> which owns this
+              The <a><code>BaseAudioContext</code></a> which owns this
               <a><code>AudioNode</code></a>.
```

Comment:
I think we do this almost everywhere.  If this is the preferred style, we need to examine the entire spec to make it consistent.  So, I'd prefer to do that separately.